docker/aws/deploy_lambda_from_s3.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from sessions import get_session
+
+account_number = sys.argv[1]
+stage = sys.argv[2]
+function_name = sys.argv[3]
+s3_bucket = sys.argv[4]
+s3_key = sys.argv[5]
+
+boto_session = get_session(account_number, "TDRJenkinsLambdaRole" + stage.capitalize())
+
+client = boto_session.client("lambda")
+
+resp = client.update_function_code(FunctionName=function_name, S3Bucket=s3_bucket, S3Key=s3_key)
```

Comment:
Does the response need to be saved to a temporary variable? It looks like it isn't used.

Reply:
You're right. I've removed it.